doc/user-guide/tcs-user-guide-generic.rst

```diff
@@ -382,40 +428,63 @@ For instance, the following sequence of commands;
 
     await tcs.offset_azel(az=10, el=0, relative=True)
     await tcs.offset_azel(az=0, el=10, relative=True)
-    await tcs.offset_azel(az=0, el=10)
+    await tcs.offset_azel(az=0, el=10, relative=False)
 
 You will result in a 10 arcsec offset in azimuth and 20 arcsec in elevation; 10 arcsec from the relative offset and 10 from the absolute offset.
 
-Furthermore, if, after doing the above, you now do;
+Combining relative and non-relative offsets can be useful if one wants to execute a pattern as shown above but want to do it with an offset with respect to the original position.
+For instance;
 
 .. code:: python
 
-    await tcs.offset_azel(az=0, el=0)
+    grid_az = [-10, +10, +10, -10]
+    grid_el = [-10, -10, +10, +10]
+
+    # offset telescope in elevation
+    await tcs.offset_azel(az=0, el=10)
+
+    # The pattern here will be entirely offset by 10 arcsec in elevation due to
+    # the command above.
+    for az,el in zip(grid_az, grid_el):
+      await tcs.offset_azel(az=az, el=el, relative=False)
+      # Do something here
+      ....
+
+    # Restore the original telescope position.
+    # Relative offset will still be in effect.
+    await tcs.offset_azel(az=0., el=0., relative=False)
+
+    # Also remove the relative offset
+    await tcs.offset_azel(az=0, el=-10)
 
-It will reset the absolute offset to zero in both azimuth and elevation but will retain the 10 arcsec relative offsets in azimuth and elevation.
 
 In all cases above, the offset will be overwritten if a new target is sent, e.g.;
```

Comment:
I feel like this needs to be in bold... or made really obvious. It might even be worth adding this near the top of the section somewhere.

Reply:
Ok, I put this phrase in bold letters and I also added a warning at the beginning of the section.